modules/SofaConstraint/src/SofaConstraint/PrecomputedConstraintCorrection.inl

```diff
@@ -739,7 +734,7 @@ void PrecomputedConstraintCorrection<DataTypes>::applyContactForce(const lineara
                 {
                     DXbuf += appCompliance[ offset2 + i ] * Fbuf[i];
                 }
```

Comment:
I guess the problem is here (i is used in two loops)

Reply:
Thanks for pointing to this direction